filters/auth/authclient.go

```diff
@@ -113,17 +113,46 @@ func createHTTPClient(timeout time.Duration, quit chan struct{}, maxIdleConns in
 		}
 	}()
 
-	return &http.Client{
-		Transport: transport,
-	}, nil
+	return transport
 }
 
-// jsonGet does a get to the url with accessToken as the bearer token in the authorization header. Writes response body into doc.
+func injectClientTrace(req *http.Request, span opentracing.Span) *http.Request {
+	trace := &httptrace.ClientTrace{
+		DNSStart: func(httptrace.DNSStartInfo) {
+			span.LogKV("dial_auth", "DNS lookup start")
```

Comment:
I have a suggestion:

Instead of putting everyr event under `dial_auth`, what do you think about having separate events denoted with `start` and `end` like `dns: start`, `dns: end`, `connect: start`, `tls_handshake: end` and so on?

Reply:
fixed